quickwit-metastore/migrations/postgresql/2021-09-02-060554_create_splits/up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE splits (
+    split_id VARCHAR(50) PRIMARY KEY,
+    split_state VARCHAR(30) NOT NULL,
+    start_time_range BIGINT,
+    end_time_range BIGINT,
+    tags TEXT NOT NULL,
+    json TEXT NOT NULL,
```

Comment:
rename `split_metadata_json`  and comment that it contains the entire split metadata.